contracts/liquidity_hub/pool-manager/src/queries.rs

```diff
@@ -1,22 +1,27 @@
 use std::cmp::Ordering;
 
 use cosmwasm_std::{Coin, Decimal256, Deps, Env, Fraction, Order, StdResult, Uint128};
-use white_whale_std::pool_manager::{SwapOperation, SwapRouteResponse};
+use white_whale_std::pool_manager::{Config, SwapOperation, SwapRouteResponse};
 use white_whale_std::pool_network::{
     asset::PairType,
     factory::NativeTokenDecimalsResponse,
     pair::{ReverseSimulationResponse, SimulationResponse},
     // router::SimulateSwapOperationsResponse,
 };
 
-use crate::state::NATIVE_TOKEN_DECIMALS;
+use crate::state::{MANAGER_CONFIG, NATIVE_TOKEN_DECIMALS};
 use crate::{
     helpers::{self, calculate_stableswap_y, StableSwapDirection},
     state::get_pair_by_identifier,
     ContractError,
 };
 use crate::{math::Decimal256Helper, state::SWAP_ROUTES};
 
+/// Query the config of the contract.
+pub fn query_config(deps: Deps) -> Result<Config, ContractError> {
+    Ok(MANAGER_CONFIG.load(deps.storage)?)
```

Comment:
given that we have several managers on V2, can we use `POOL_MANAGER_CONFIG` to make it super clear? Or just `CONFIG`

Reply:
yeah maybe just config could be. I remember at first we started naming them `MANAGER_CONFIG` cuz maybe we could have a config for the pools, or vaults... but that wasn't the case. I think we can refactor that later once all the contracts are ready.